pkg/auditlog/client.go

```diff
@@ -67,9 +66,12 @@ func (c *Client) Publish(ctx context.Context, e Event) error {
 	}
 	req = req.WithContext(ctx)
 	req.Header.Set("Content-Type", "application/json")
-	req.Header.Set("Idempotency-Key", e.IdempotencyKey)
 	req.Header.Set("Authorization", "Bearer "+c.APIKey)
 
+	if len(e.IdempotencyKey) > 0 {
```

Comment:
little style stuff:
use 
```go
e.IdempotencyKey != ""
```
to compare string. The compiler does the optimization by himself and when you read the code you know directly that we deal with a string.

Use the `len(xx) != 0` when you check a slice or map length.

Reply:
That's good to know!